src/csharp/Microsoft.Spark/Sql/Types/SimpleTypes.cs

```diff
@@ -119,16 +119,38 @@ public sealed class ShortType : IntegralType
     }
 
     /// <summary>
-    /// Represents a decimal type (not implemented).
+    /// Represents a decimal type.
     /// </summary>
     public sealed class DecimalType : FractionalType
     {
+        internal static Regex s_fixedDecimal =
+            new Regex(@"decimal\s*\((\d+),\s*(\d+)\)", RegexOptions.Compiled);
```

Comment:
Are the \s* required ?

PySpark's spark/python/pyspark/sql/types.py:
```python
    def jsonValue(self):
        return "decimal(%d,%d)" % (self.precision, self.scale)
```

And the below:
```csharp
        public override string SimpleString => $"decimal({_precision},{_scale})";
```

doesn't contain spaces.

Reply:
Yes. The true source should be https://github.com/apache/spark/blob/1e87694f2bc0dea3eaaef936dc8552b3ba421539/sql/catalyst/src/main/scala/org/apache/spark/sql/types/DataType.scala#L117

Reply:
Let's add the missing \s* after the (\d+) capture groups.

Reply:
Sure. Let me change that.